operator/pkg/util/k8s.go

```diff
@@ -61,3 +65,25 @@ func DetectSupportedJWTPolicy(config *rest.Config) (JWTPolicy, error) {
 	}
 	return FirstPartyJWT, nil
 }
+
+// GetGvkString differs from default representation. Each one
+// is separated by slashes.
+func GetGvkString(gvk schema.GroupVersionKind) string {
```

Comment:
nit: s/Gvk/GVK/
also, Foo() is usually preferred in Go to GetFoo() unless there's a very good reason to include the Get part.